config/jobs/kubernetes-sigs/cluster-api-provider-ibmcloud/cluster-api-provider-ibmcom-presubmits-release-0.12.yaml

```diff
@@ -0,0 +1,170 @@
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-ibmcloud
+      testgrid-tab-name: pr-make-release-0-12
+  - name: pull-cluster-api-provider-ibmcloud-test-release-0-12
+    cluster: eks-prow-build-cluster
+    always_run: true
+    branches:
+      # The script this job runs is not in all branches.
+      - ^release-0.12
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-ibmcloud
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250904-c89b045f57-1.33
+        imagePullPolicy: Always
+        command:
+        - "./scripts/ci-test.sh"
+        resources:
+          limits:
+            cpu: "2"
+            memory: "6Gi"
+          requests:
+            cpu: "2"
+            memory: "6Gi"
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-ibmcloud
+      testgrid-tab-name: pr-test-release-0-12
+  - name: pull-cluster-api-provider-ibmcloud-smoke-test-release-0-12
+    cluster: eks-prow-build-cluster
+    always_run: true
+    branches:
+      # The script this job runs is not in all branches.
+      - ^release-0.12
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-ibmcloud
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250904-c89b045f57-1.33
+        imagePullPolicy: Always
+        env:
+        - name: "IBMCLOUD_API_KEY"
+          value: "dummyApiKey"
+        command:
+        - "runner.sh"
+        - "./scripts/ci-smoke-test.sh"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          limits:
+            cpu: "2"
+            memory: "6Gi"
+          requests:
+            cpu: "2"
+            memory: "6Gi"
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-ibmcloud
+      testgrid-tab-name: pr-smoke-test-release-0-12
+  - name: pull-cluster-api-provider-ibmcloud-build-release-0-12
+    cluster: eks-prow-build-cluster
+    always_run: true
+    branches:
+      # The script this job runs is not in all branches.
+      - ^release-0.12
+    optional: false
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-ibmcloud
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250904-c89b045f57-1.33
+        command:
+        - "./scripts/ci-build.sh"
+        resources:
+          limits:
+            memory: "6Gi"
+            cpu: "2"
+          requests:
+            memory: "6Gi"
+            cpu: "2"
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-ibmcloud
+      testgrid-tab-name: pr-build-release-0-12
+  - name: pull-cluster-api-provider-ibmcloud-verify-release-0-12
```

Comment:
@Prajyot-Parab, as we recently merged the checks for links and yaml lint in `make verify` which needs DIND enabled, this would need additional configuration similar to the main branch, PTAL - https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-provider-ibmcloud/cluster-api-provider-ibmcom-presubmits.yaml#L119-L149